organisaatio-ui/src/main/webapp/js/init.js

```diff
@@ -51,24 +51,30 @@ function organisaatioInitialize() {
 
     // Make sure this function is not spammed.
     var lang = 'fi';
-    var promiseCasMe = $http.get(CAS_ME_URL).then(
-        function(response) {
-            console.log("Success on " + CAS_ME_URL, response);
-            lang = response.data.lang;
-            if (lang) {
-                // Toistaiseksi vain SV on tuettu FI:n lisäksi
-                lang = (lang==="sv" ? "sv" : "fi");
-            } else {
-                console.warn('failed parsing result, defaulting to fi');
-            }
-            window.APP_CAS_ME = response.data;
-            window.APP_CAS_ME.lang = lang.toLowerCase();
-        },
-        function(response) {
-            console.warn("Failed to get: " + CAS_ME_URL + " --> using language: " + lang);
-            window.APP_CAS_ME = {};
-            window.APP_CAS_ME.lang = lang.toLowerCase();
-    });
+    var failureHandler = function (response) {
+      console.warn(
+        "Failed to get: " + CAS_ME_URL + " --> using language: " + lang
+      );
+      window.APP_CAS_ME = {};
+      window.APP_CAS_ME.lang = lang.toLowerCase();
+    };
+    var promiseCasMe = $http
+      .get(CAS_ME_URL)
+      .then(function () {
+        return $http.get(CAS_ME_URL);
+      }, failureHandler)
+      .then(function (response) {
+        console.log("Success on " + CAS_ME_URL, response);
+        lang = response.data.lang || lang;
```

Comment:
this is where it broke down, added failsafe value even though it's not needed anymore